crates/hyperswitch_domain_models/src/errors/api_error_response.rs

```diff
@@ -0,0 +1,637 @@
+#![allow(dead_code, unused_variables)]
```

Comment:
Can you check if these two warnings need to be allowed, or if we can remove this annotation?